test/multi-table.wast

```diff
@@ -25,7 +28,11 @@
   (elem $empty func)
   (elem $declarative declare func $h)
 
+  ;; This elem will be emitted as usesExpressions because of the type of the
+  ;; table.
+  (elem $especial (table $tspecial) (i32.const 0) (ref null $none_=>_none) $f $h)
```

Comment:
Hmm, I don't think we should be able to parse this format that uses function names and a specific type. The only allowable use of function indices or names is with "func" and it's always short for "funcref"

https://webassembly.github.io/spec/core/text/modules.html#:~:text=also%2C%20the%20element%20list%20may%20be%20written%20as%20just%20a%20sequence%20of%20function%20indices%3A

Reply:
Hmm, you might be right, based on that spec fragment. However, the reference spec interpreter supports a bunch of extra sugar - is all that documented in the spec?

Separately, I'm not sure there is harm in reading this, since it actually simplifies our code to do so (a single loop) and we aren't really in the business of parsing the spec format with 100% precision anyhow.